scripts/wawCommons.py

```diff
@@ -411,9 +411,21 @@ def getParametersCombination(config, *args):
     
     return parametersCombinationMap
 
-def setLoggerConfig():
-    fileConfig(os.path.split(os.path.abspath(__file__))[0]+'/logging_config.ini')
+def verbose(self, message, *args, **kws):
+    if (logging.Logger.isVerbose):
+        self._log(logging.INFO, message, args, **kws)
```

Comment:
I'm not sure, but I think that it should be `*args` (without that we pass just array that will be taken as a single argument of calling function).

Reply:
In this case passing the arguments as an array is really correct. The reason is the way the `self._log()` function is implemented. But I changed it to `self.info()` anyway for better understandability.